### ISSUE TITLE
Connection problem with username string too long

### ISSUE THREAD

**Issue by SilverBull** [h3imdall/ftp-remote-edit#283]
Hello, I encountered a strange problem with username length, I noticed that one of my ftp instance doesn't work on the plugin but work just fine with Filezilla, while another instance have no problem whatsoever, after the routine controls on credential I noticed that the only discriminant was the username length because my host (Namecheap) use user@subdomain as login.
I did some testing and isolated the problem, it appears that login longer than 32 character result in a "read ECONNRESET" error

TL DR: username longer than 32 character cause "read ECONNRESET"

**Comment by cschindl** [comment]
Hi @SilverBull,
 
unfortunately I could not reproduce the error locally. I have a Filezilla server running locally and use testuser@verylonglonglonglonglonlonglongusername.com for both username and password. It is more than 32 characters long and works fine. "ECONNRESET" means the other side of the conversation abruptly closed its end of the connection. This is most probably due to one or more application protocol errors. 

Can you make a screenshot of the FTP log (located as one line below the server list and can be expanded, see screenshot) and upload it here so I can see which FTP commands are successful and which are causing this error? 

![image](https://user-images.githubusercontent.com/28109094/56227177-233ab180-6075-11e9-9a51-79f2243ce400.png)

**Comment by SilverBull** [comment]
Hi @cschindl I tried getting the log but as you can see on the problematic one I don't get any response at all, instead on the other one I log in normally, I isolated the char problem creating another subdomain with a short name and changing the username until I wasn't able to login anymore, since you don't have the problem, it could be a namecheap bug? What else can I do to help you?
![atom_2019-04-16_19-02-24](https://user-images.githubusercontent.com/12399837/56229493-4fa4fc80-607a-11e9-965d-3d51c1ca610f.png)
![atom_2019-04-16_19-01-40](https://user-images.githubusercontent.com/12399837/56229494-4fa4fc80-607a-11e9-86a8-3f44be1b9fcc.png)

**Comment by cschindl** [comment]
I'm afraid I can't reproduce the error. Therefore it seems to be a problem of the hoster. Pure-FTP servers cause the biggest problems until now. If you can freely choose your username, then you can create one where the problem does not occur. Without test access and without error-reproduction I can't help you. Sorry.

**Comment by SilverBull** [comment]
Understandable, I will try to create subdomains and usernames under 32 characters. The fact is that filezilla works without hassle but this plugin is way more comfortable to work with. Thank you for your time
